### PR TITLE
Upgrade for breaking changes in rippled protocol buffers

### PR DESCRIPTION
**Pull request by keefertaylor**
## High Level Overview of Change

Update to be compatible with breaking protocol buffer changes in rippled.  Breaking changes were introduced in  https://github.com/ripple/rippled/pull/3254. .

### Context of Change

No logic changes are introduced. This change does not touch the public API. 

Changes for protocol buffer compatibility and upgrade xpring-common-js to a version that has compatibility wiht the new protocol buffers

Changes in the protocol buffers are:
- Change folder / package location of files
- Use numeric strings instead of numbers to prevent overflows when working with 64 bit integers.
- Introduce new layers of abstraction internally.

### Type of Change

- [x] Refactor (non-breaking change that only restructures code)

## Before / After

Compatible with latest rippled. 

## Test Plan
 CI

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -18,5 +18,7 @@
     },
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
-    "files.trimTrailingWhitespace": true
+    "files.trimTrailingWhitespace": true,
+
+    "mochaExplorer.files": "test/*.ts"
```
Comment (Stormtv) [inline .vscode/settings.json]:
```suggestion
    "mocha.files.glob": "test/**/*-test.ts",
    "mocha.requires": [
        "ts-node/register"
    ]
```
This fixed the sidebar for me

File: src/default-xpring-client.ts [hunk 2]
```diff
@@ -143,39 +159,54 @@ class DefaultXpringClient implements XpringClientDecorator {
...
+    destinationAccountAddress.setAddress(destinationAddress)
+
+    const destination = new Destination()
+    destination.setValue(destinationAccountAddress)
 
     const account = new AccountAddress()
     account.setAddress(sender.getAddress())
 
+    const account2 = new Account()
+    account2.setValue(account)
```
Comment (Stormtv) [inline src/default-xpring-client.ts]: Better name for this, or at least explain why we have to do this?

File: src/default-xpring-client.ts [hunk 3]
```diff
@@ -143,39 +159,54 @@ class DefaultXpringClient implements XpringClientDecorator {
...
+      lastValidatedLedgerSequence + maxLedgerVersionOffset,
+    )
+
+    const signingPublicKeyBytes = Utils.toBytes(sender.getPublicKey())
+    const signingPublicKey = new SigningPublicKey()
+    signingPublicKey.setValue(signingPublicKeyBytes)
+
     const transaction = new Transaction()
-    transaction.setAccount(account)
+    transaction.setAccount(account2)
```
Comment (hbergren) [inline src/default-xpring-client.ts]: What is `account2` here? Why did this switch from `account` to `account2`? What is `account`?

Misc: Can we name `account2` something more descriptive?

Comment (keefertaylor) [reply src/default-xpring-client.ts]: Yeah, sorry about that. leftover code from prototyping.

**Pull request merged**